### DISCUSSION TITLE
Report C2039: "_snprintf" is not std members while Complie json.hpp with boost library

### DISCUSSION THREAD

**Discussion by Morgenlin** [nlohmann/json#2609] [Q&A] [answered]
hello:
              I created a QT project recently. But there is a problem bothers me in days. I was using nlohmann/json.hpp to write some config files, its been work smoothly. The other day, I added up Boost library in my project, then there was an error: "C2039: "_snprintf" is not std members in json.hpp file!", I tried different Boost library release like 1_70、1_74 and different json release. They were all have same error.
             I was wondering if anyone have been under this situation, And How did you solve it?
             Thank you!

**Comment by nlohmann** [comment, marked as answer]
Maybe related:

- https://github.com/nlohmann/json/issues/1437
- https://github.com/nlohmann/json/issues/1408

**Reply by nlohmann** [reply]
Do you need further assistance with this issue?
